.travis.yml

```diff
@@ -19,10 +19,10 @@ cache:
 
 python:
   - "2.7"
-  - "3.5"
   - "3.6"
   - "3.7"
   - "3.8"
+  - "3.9"
```

Comment:
Not available in travis yet, but I expect it will be in a matter of days. So, better this than `3.9-dev`